build/copy-schemas.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Utility for copying schemas to their dist folders.
+ * Usage: node build/copy-schemas.js
+ */
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+
+const rootDir = path.resolve(process.cwd());
+const srcSchemaPaths = "src/**/*.schema.json";
+const destDir = "dist";
```

Comment:
should this also be "dest"?

Reply:
no its for the dist folder for our distribution